react-components/src/utilities/queryKeys.ts

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type IdEither } from '@cognite/sdk/';
+
+export const queryKeys = {
+  all: ['cdf'] as const,
+  // ASSETS
+  assets: () => [...queryKeys.all, 'assets'] as const,
+  assetsById: (ids: IdEither[]) => [...queryKeys.all, 'assets', ids] as const,
```

Comment:
Is `assetsById` supposed to use the `assets` query key instead of `queryKeys.all`?

Reply:
Ops, make sense.
https://github.com/cognitedata/reveal/pull/4414/commits/c9a50e52b105e8f50cceda82d6fe1e4d14b2dfe7